### DISCUSSION TITLE
Looking for first issues to work on

### DISCUSSION THREAD

**Discussion by John42506176Linux** [alan-turing-institute/sktime#719] [Q&A]
Hello, currently trying to start contributing and I was having a hard time finding a first issue to start working on, as it looks most of them are already being worked on at the moment. I was wondering if anyone could point me in the right direction here.

Thanks,

**Comment by fkiraly** [comment]
Giving some general advice in case of interest to others too.

[update: We now have created a "good first issues" collection issue here:
#1147 
this also includes "getting started with contributions" advice]

First entry point is typically looking at issues with "good first issue" tag.
You can "take" them by writing a short statement of commitment in there.

If you see an issue that looks interesting and taken, but last activity is a while ago, it could be (and it happens often) that the contributor there in the end didn't have the time but forgot to "uncommit" from the issue. In that case, you can post a quick message to see whether the person is still working on it; if no reply within 1-2 weeks (and no active pull request linked to it), assume it's no longer worked on.

Other ways to find "good first issues":
* look at roadmap and "wish list" issues. The "wish lists of the day" are pinned at the top of the issue tracker. You can find items there that may need some design or are not entirely explicit.
* if still unsure, you can directly contact core developers - you can do this via gitter or slack to the community; or, you can ping cc members and ask (currently @fkiraly, @mloning, @TonyBagnall)

**Comment by fkiraly** [comment]
We now have created a "good first issues" collection issue here:
https://github.com/alan-turing-institute/sktime/issues/1147

Have a look, if still interested, @John42506176Linux.
